README.md

```diff
@@ -1 +1,63 @@
-# PetStoreAPITesting
\ No newline at end of file
+# Pet Store API Automation Testing
+
+## Portfolio project on automation of test scripts using API tests for [Swagger Pet Store](https://petstore.swagger.io/#/)
+
+## Contents:
+
+- [Technology stack used](https://github.com/DaryaAndreyuk/PetStoreAPITesting/tree/fb_DaryaAndreyuk_8_AddReadMe?tab=readme-ov-file#technology-stack-used)
+- [Running the Tests from the terminal](https://github.com/DaryaAndreyuk/PetStoreAPITesting/tree/fb_DaryaAndreyuk_8_AddReadMe?tab=readme-ov-file#running-the-tests)
+- [Viewing Allure Reports](https://github.com/DaryaAndreyuk/PetStoreAPITesting/tree/fb_DaryaAndreyuk_8_AddReadMe?tab=readme-ov-file#viewing-allure-reports)
+- [Build with Jenkins](https://github.com/DaryaAndreyuk/PetStoreAPITesting/tree/fb_DaryaAndreyuk_8_AddReadMe?tab=readme-ov-file#build-with-jenkins)
+
+## Technology stack used
+
+<p align="center" dir="auto">
+<a href="https://www.jetbrains.com/idea/" rel="nofollow"><img width="11%" title="IntelliJ IDEA" src="images/Intelij_IDEA.svg" alt="Intellij_IDEA" style="max-width: 100%;"></a>
+<a href="https://www.java.com/" rel="nofollow"><img width="11%" title="Java" src="images/Java.svg" alt="Java" style="max-width: 100%;"></a>
+<a href="https://allurereport.org/" rel="nofollow"><img width="11%" title="Allure Report" src="images/Allure_Report.svg" alt="Allure_Report" style="max-width: 100%;"></a>
+<a href="https://gradle.org/" rel="nofollow"><img width="11%" title="Gradle" src="images/Gradle.svg" alt="Gradle" style="max-width: 100%;"></a>
+<a href="https://www.jenkins.io/" rel="nofollow"><img width="11%" title="Jenkins" src="images/Jenkins.svg" alt="Jenkins" style="max-width: 100%;"></a>
+<a href="https://github.com/"><img width="12%" title="GitHub" src="images/GitHub.svg" alt="GitHub" style="max-width: 100%;"></a>
+<a href="https://rest-assured.io/" rel="nofollow"><img width="11%" title="Rest Assured" src="images/RestAssured.svg" alt="RestAssured" style="max-width: 100%;"></a>
+<a href="https://testng.org/" rel="nofollow"><img width="12%" title="TestNG" src="images/TestNG.png" alt="TestNG" style="max-width: 100%;"></a>
+
+</p>
+
+In this project, automated tests are written in `Java` using the `Rest-Assured` framework for API testing.
```

Comment:
Which version of Java? It's could be important

Reply:
@topsycreed fixed, thank you